gocd/vagrant-publisher.gocd.yaml

```diff
@@ -0,0 +1,22 @@
+format_version: 3
+pipelines:
+  Vagrant.Publish:
+    group: Vagrant
+    lock_behavior: unlockWhenFinished
+    environment_variables:
+      ATLAS_TOKEN: '{{SECRET:[opensuse.secrets][ATLAS_TOKEN]}}'
+    materials:
+      git:
+        git: https://github.com/openSUSE/opensuse-release-tools.git
+    timer:
+      spec: 0 0 0 ? * *
+      only_on_changes: false
+    stages:
+    - Run:
+        approval: manual
+        resources:
+        - obs-to-vagrantcloud
+        tasks:
+        - script: |
+            ruby.ruby3.0 obs-to-vagrantcloud.rb --url https://download.opensuse.org/tumbleweed/appliances/boxes/Tumbleweed.x86_64.json --organization opensuse --new-box-name Tumbleweed.x86_64 --provider libvirt
```

Comment:
Is it necessary to hardcode the version here? This way it needs to be updated on every container update with newer minor version of ruby...

Reply:
Sadly, yes :-(

The dependencies of obs-to-vagrantcloud only exist for Ruby 3.0, but `/usr/bin/ruby` is ruby 2.7 in Tumbleweed and the ruby interpreter is not managed by alternatives…